<p>
The &lt;strong&gt;/&lt;b&gt; and &lt;em&gt;/&lt;i&gt; tags have exactly the same effect in most web browsers, but there is a fundamental difference between them: &lt;strong&gt; and &lt;em&gt; have a semantic meaning whereas &lt;b&gt; and &lt;i&gt; only convey styling information like CSS.
</p>

<p>
When &lt;b&gt; can have simply no effect on a Palm device with limited display or when a screen reader software is used by a blind person, &lt;strong&gt; will:
</p>

<ul>
  <li>Underline the characters on a Palm Pilot</li>
  <li>Speak with lower tone when using a screen reader such as Jaws</li>
  <li>And display the text bold in normal browsers</li>
</ul>

<p>Consequently:</p>

<ul>
  <li>in order to convey semantics, the &lt;b&gt; and &lt;i&gt; tags shall never be used,</li>
  <li>in order to convey styling information, the &lt;b&gt; and &lt;i&gt; should be avoided and CSS should be used instead.</li>
</ul>

<h2>Noncompliant Code Example</h2>

<pre>
&lt;i&gt;car&lt;/i&gt;                 &lt;!-- Non-compliant --&gt;
&lt;b&gt;train&lt;/b&gt;               &lt;!-- Non-compliant --&gt;
</pre>

<h2>Compliant Solution</h2>

<pre>
&lt;em&gt;car&lt;/em&gt;               &lt;!-- Compliant --&gt;
&lt;strong&gt;train&lt;/strong&gt;     &lt;!-- Compliant --&gt;
</pre>
